machineconfiguration/v1alpha1/0000_80_machineconfigstate.crd.yaml

```diff
@@ -0,0 +1,185 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: machineconfigstates.machineconfiguration.openshift.io
+  labels:
+    "openshift.io/operator-managed": ""
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  # group name to use for REST API: /apis/<group>/<version>
+  group: machineconfiguration.openshift.io
+  scope: Cluster
+  names:
+    kind: MachineConfigState
+    singular: machineconfigstate
+    plural: machineconfigstates
+    shortNames:
+      - mcs
```

Comment:
In general we don't add shortname for configuration objects like this where there are only going to be a handful in the cluster and in particular where administrators are the only people interacting with them

Shortnames cause too much likelihood of collision, and techincally oc only actually supports fully qualified names. Therefore we try to reserve shortnames for daily access, by many users, style resources

Reply:
not even something like `mcstate` or if we change the name `mcnodes`? Not a blocker, just trying to take every step to make this easier to adopt.

Reply:
No short names, sorry